red5pro/streaming-html5
# Feature/tcp transport support rpro 5262

The ability to select `UDP` or `TCP` ICE transport is added to the configuration/settings page so that all tests can take advantage of the configuration setting.

**To Verify:**
When making a Publish offer, you should see something similar printed in the console (note the `transport` attribute):

```js
[red5pro-sdk] debug: (R5ProPublisherSocket) [websocket-post]: {
  "handleOffer": "stream1",
  "transport": "tcp",
  "data": {
    "sdp": {
      "type": "offer",
      "sdp": "..."
    }
  }
}
```

The Subscriber request of offer will have a `transport` attribute as well:

```js
 {
  requestOffer: streamName,
  requestId: subscriptionId,
  transport: transport
}
```

**You may need to clear you Session DB in the browser**

To do so:

1. Open up the Dev Console.
2. Select `Application`
3. Expand `Storage > Session Storage`
4. Right-Click on the url
5. Select `Clear`
